pre_commit/commands/run.py

```diff
@@ -224,7 +224,14 @@ def _run_hooks(config, hooks, args, environ):
                 '`pre-commit install`.',
             )
         output.write_line('All changes made by hooks:')
-        subprocess.call(('git', '--no-pager', 'diff', '--no-ext-diff'))
+        if args.color:
+            subprocess.call((
+                'git', '--no-pager', 'diff', '--no-ext-diff',
+                '--color={}'.format(args.color),
+            ))
+        else:
+            subprocess.call(('git', '--no-pager', 'diff', '--no-ext-diff'))
```

Comment:
I think basically what we're going to have to end up with is:

```python
cmd = ('git', '--no-pager', 'diff', '--no-ext-diff')
if args.color:
    cmd += ('--color=always',)
subprocess.call(cmd)
```

Since at this point `args.color` will always be a boolean